product-service/handlers/getProductsById/getProductsById.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { withCorsHeaders } from "../../utils/withCorsHeaders";
+import { getProductsById } from ".";
+
+const mockProductList = [
+  {
+    count: 1,
+    description: "Description",
+    id: "1",
+    price: 10,
+    title: "Toy",
+    src: "url",
+  },
+];
+
+jest.requireActual("node-fetch");
+jest.mock("../../data/productList.json", () => [
+  {
+    count: 1,
+    description: "Description",
+    id: "1",
+    price: 10,
+    title: "Toy",
+    src: "url",
+  },
+]);
+
+describe("#getProductsById", () => {
+  it("should return product if it exists", async () => {
+    const targetProduct = mockProductList[0];
+    delete targetProduct.count;
```

Comment:
Why do you need to delete count?

Reply:
I suppose this is because of this?
```js
    const randomCounts = await fetch(RANDOM_NUMBER_API).then((res) =>
      res.json()
    );
```

Reply:
Because `getProductsById` sets random count number using [RANDOM_NUMBER_API](https://github.com/svetlanagorchanina/nodejs-aws-be/blob/3831b6dbc1cda071698f61edfed147a686edbc2a/product-service/handlers/getProductsById/index.ts#L6)